dockers/docker-orchagent/swssconfig.sh

```diff
@@ -41,6 +41,22 @@ fast_reboot
 
 HWSKU=`sonic-cfggen -d -v "DEVICE_METADATA['localhost']['hwsku']"`
 
+# Don't load json config if system warm start or
+# swss docker warm start is enabled, the data already exists in appDB.
+WARM_START=`redis-cli -n 4 hget "WARM_RESTART|system" enable`
+if [ "$WARM_START" == "true" ]; then
+  exit 0
+fi
+
+WARM_START=`redis-cli -n 4 hget "WARM_RESTART|swss" enable`
+if [ "$WARM_START" == "true" ]; then
+  # We have to make sure db data has not been flushed.
+  WARM_START=`redis-cli -n 6 hget "WARM_RESTART_TABLE|orchagent" restart_count`
+    if [ -n "$WARM_START" ] && [ "$WARM_START" != "0" ]; then
+      exit 0
+    fi
```

Comment:
>fi [](start = 4, length = 2)

else, should we treat it as 'logic error' and return error code? Your code fallbacks to cold start, which may hide bugs in other components.

Reply:
This is to handle the case of system cold boot while swss docker warm enabled. 
swss docker warm restart knob is only supposed to handle the scenario of individual docker warm restart.